vllm/distributed/device_communicators/custom_all_reduce_utils.py

```diff
@@ -219,9 +218,13 @@ def gpu_p2p_access_check(src: int, tgt: int) -> bool:
         with tempfile.NamedTemporaryFile() as output_file:
             input_bytes = pickle.dumps(
                 (batch_src, batch_tgt, output_file.name))
+            # TODO(T212433680) make this sync'able
+            import libfb.py.parutil
             returned = subprocess.run([sys.executable, __file__],
                                       input=input_bytes,
-                                      capture_output=True)
+                                      capture_output=True,
+                                      env={**os.environ, "PYTHONPATH": libfb.py.parutil.get_runtime_path()})
```

Comment:
I thought the env var in subprocess should inherit the current process by default.

Reply:
nope, internal packing / build system may not set this correctly.

Even in PyTorch, we need such mechanism to override some functions, such as https://github.com/pytorch/pytorch/blob/main/torch/_utils_internal.py#L61, which we have an internal version of _utils_internal.py

Reply:
Is it possible to frame it as a general problem? e.g. we can solve it under the context that some users use packed Python, and does not work with vllm.

vllm is happy to make changes to accomodate to the usage, but replacing files sounds too hacky to me.